CONTRIBUTING.md

```diff
@@ -83,9 +83,11 @@ $ docker rm simple-icons-website-dev
 ### How to add new locales
 
 1. Add the languages to the file *locales/languages.json*.
-2. Run `npm run locale` to generate the new locale files. These will be `.po` extension files located in the *locales/* folder.
-3. Translate the strings in the new locale files and will be automatically included in the next bundle.
+2. Run `npm install` in the root directory to install all required dependencies, if you haven't done so already.
+3. Run `npm run locale` in the root directory to generate the new locale files. These will be `.po` extension files located in the *locales/* folder.
```

Comment:
It's unnecessary to run commands in the root directory. You can run the command anywhere in the project.